modules/porous_flow/src/materials/PorousFlowMaterialPermeabilityUnity.C

```diff
@@ -0,0 +1,32 @@
+/****************************************************************/
+/* MOOSE - Multiphysics Object Oriented Simulation Environment  */
+/*                                                              */
+/*          All contents are licensed under LGPL V2.1           */
+/*             See LICENSE for full restrictions                */
+/****************************************************************/
+
+#include "PorousFlowMaterialPermeabilityUnity.h"
```

Comment:
Oh, good.  Thanks for doing this "Unity" version.  I didn't see it before i made the comments above.  Remove "Material" from the name.  Also, i don't think it should be registered in PorousFlowApp as i think it's very unlikely anyone will use it in real simulations.  But, if you disagree just register it, no worries.